brain/circuit.cpp

```diff
@@ -98,19 +101,15 @@ neuron::Morphologies Circuit::loadMorphologies( const GIDSet& gids,
     GIDSet::const_iterator gid = gids.begin();
     for( size_t i = 0; i < uris.size(); ++i, ++gid )
```

Comment:
Up to @eile to decide, I tend to avoid increments inside the loop body. On the other hand: auto gid = gids.begin(); is welcome.